### PR TITLE
Edits to CLI reference

### PR DESCRIPTION
**Pull request by mdlinville**
A couple small edits I noticed when reviewing https://github.com/docker/docker.github.io/pull/2633

Needs to be cherry-picked to 17.03 and 17.04, please.

### CODE REVIEW

File: docs/reference/commandline/network.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ Run 'docker network COMMAND --help' for more information on a command.
 
 ## Description
 
-Manage networks. You can use subcommand to create, list, inspect, remove,
-connect and disconnect networks.
+Manage networks. You can use subcommands to create, inspect, list, remove,
+prune, connect and disconnect networks.
```
Comment (thaJeztah) [inline docs/reference/commandline/network.md]: `, and disconnect`?

**Review by thaJeztah — COMMENTED** [review]
one tiny comment, but LGTM otherwise

**Comment by mstanleyjones** [comment]
Addressed.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**